ws-nextjs-app/cypress/e2e/articlePage/index.cy.ts

```diff
@@ -251,6 +299,33 @@ const atiAnalyticsTestSuites = [
       assertContinueReadingButtonComponentView,
     ],
   },
+  {
+    path: '/gahuza/articles/c5y51yxeg53o',
+    runforEnv: ['local'],
+    service: 'gahuza',
+    pageIdentifier: 'gahuza.articles.c5y51yxeg53o.page',
+    siteId: 40,
+    applicationType: 'responsive',
+    contentType: 'article',
+    useReverb: true,
+    tests: [
+      assertPageView,
+      assertArticleLiteSiteLinkComponentView,
+      assertArticleLiteSiteLinkComponentClick,
+      assertFeaturesAnalysisComponentView,
+      assertFeaturesAnalysisComponentClick,
+      assertMostReadComponentView,
+      assertMostReadComponentClick,
+      assertPodcastPromoComponentView,
+      assertPodcastPromoComponentClick,
+      assertRelatedTopicsComponentView,
+      assertRelatedTopicsComponentClick,
+      assertRelatedContentComponentView,
+      assertRelatedContentComponentClick,
+      assertTopStoriesComponentView,
+      assertTopStoriesComponentClick,
```

Comment:
Do some of these need excluded if its the Lite version being requested?

I can see we have this logic currently in the `specialFeatures/atiAnalytics/` tests: https://github.com/bbc/simorgh/blob/0ee15e31cbd4d7510c54b87df11f25b91d6e85d3/ws-nextjs-app/cypress/e2e/specialFeatures/atiAnalytics/index.cy.ts#L296-L309